src/common.ts

```diff
@@ -13,20 +13,37 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {google as databaseAdmin} from '../proto/spanner_database_admin';
+
 import {ServiceError, CallOptions} from 'grpc';
+import {Operation as GaxOperation} from 'google-gax';
+import {google as instanceAdmin} from '../proto/spanner_instance_admin';
+import {google as databaseAdmin} from '../proto/spanner_database_admin';
+
+export type IOperation = instanceAdmin.longrunning.IOperation;
+export interface TransactionOptions {
```

Comment:
Do we need this? I think `readOnly` should be gone and we have [`TimestampBounds`](https://github.com/googleapis/nodejs-spanner/blob/master/src/transaction.ts#L42)

Reply:
my bad, Removed!!